content/docs/user-guide/basic-concepts/run-cache.md

```diff
@@ -0,0 +1,11 @@
+---
+name: 'Run-cache'
+match: ['run-cache']
+---
+
+The DVC run-cache is a log of stage runs in the project. It's comprised of
+`dvc.lock` file backups, identified as combinations of dependencies, commands,
+and outputs that correspond to each other. `dvc repro` and `dvc run` populate
```

Comment:
> `dvc.lock` file backups, identified as combinations of dependencies, commands, and outputs

Is that the exact formula that makes up the combinations that identify runs, or is there more to it?

Reply:
Not the exact formula, but that's what it consists of. I don't think it is worth going that deep in that doc.

Reply:
Like I mentioned I'm just curious about the hash algo itself. But I did want to confirm that those are all of the variables of the formula.